cookbooks/cdo-varnish/test/shared/shared.rb

```diff
@@ -96,7 +96,7 @@ def escape(s)
 
       def _request(url, headers={}, cookies={}, method='GET')
         header_string = headers.map { |key, value| "-H \"#{key}: #{value}\"" }.join(' ')
-        cookie_string = cookies.empty? ? '' : "--cookie \"#{cookies.map{|k,v| "#{escape(k)}=#{escape(v)}"}.join('; ')}\""
+        cookie_string = cookies.empty? ? '' : "--cookie \"#{cookies.map{|k, v| "#{escape(k)}=#{escape(v)}"}.join('; ')}\""
```

Comment:
This is a common paradigm (snippet) across our codebase. Do we prefer the right? Is this reason not to enable this cop?